proseco/characteristics.py

```diff
@@ -103,25 +109,16 @@ def __getattr__(name):
 def _set_aca_limits():
     """Set global variables for ACA thermal planning and penalty limits"""
     global chandra_models_version
-    from xija.get_model_spec import get_xija_model_spec
 
-    spec, chandra_models_version = get_xija_model_spec(
-        "aca", version=chandra_models_version
-    )
+    spec_txt, info = chandra_models.get_data("chandra_models/xija/aca/aca_spec.json")
```

Comment:
I'd assumed the plan was to update get_xija_model_spec file to use chandra_models.get_data to have a unified approach via that strategy.

Reply:
Yes, but that is out of scope for this PR. Also, using the direct interface in `ska_helpers` makes more sense here since there is no real use of `xija` itself.

Reply:
Yes -- though for me I was bringing it up because starcheck was getting the ACA model version and the limits from proseco here via characteristics.py, but getting the aca model spec with get_xija_model_spec.  That leads to a disconnect in that application due to the interface change here (in scope).  It probably makes sense for starcheck to either join proseco in using ska_helpers.chandra_models.get_data to get the aca model or to stop using proseco.characteristics as a convenient source of limit reading.  Or update get_xija_model_spec to use ska_helpers.chandra_models.get_data.

Reply:
I should mention that `get_xija_model_spec()` does not work on FOT Windows machines.  it returns an error about not being able to read from remote repository when executing the line `repo = git.Repo.clone_from(repo_path, repo_path_local)`.  I have confirmed that this occurs from the python environment directly, and is not related to the pyexec interface.  I think we don't run into this issue now only because MATLAB provides all the necessary inputs, so this code doesn't run when called from MATLAB.  Except it did in my testing of removing this parameter (before I started using this PR).

Reply:
Though James I was working under the impression that part of the problem with get_xija_model_spec was that right now the windows environment had a broken git .

Reply:
We had previously seen what we guessed was a conda bug making a broken git.  It was a non-issue because the windows environment wasn't using git for anything but will hope it is fixed upstream or will work around it for the next testing release (if that is coming into play).